docs/en/credential-issuance-l2plus.rst

```diff
@@ -0,0 +1,671 @@
+     - REQUIRED. Challenge data for cryptographic document validation. It SHOULD be the SHA-256 hash value of a random value with the ``auth_session`` and timestamp for cryptographic binding.
+   * - **mrtd_pop_nonce**
+     - string
+     - REQUIRED. Unique nonce value for the next step, preventing replay attacks.
+   * - **mrz**
+     - string
+     - OPTIONAL. Machine Readable Zone information including document number, date of birth, expiry date, citizenship and gender.
+
+Below a non-normative example of an MRTD PoP Response:
+
+.. code-block:: http
+
+    HTTP/1.1 202 Accepted
+    Content-Type: application/json; charset=utf-8
+
+    {
+      "mrtd_pop_nonce":"9f2c4a7e3b1d8c9a6e5f4b2a1c3d7e8f",
+      "mrz":"...",
+      "challenge":"..."
+    }
+
+**The MRTD PoP Service MUST:**
+
+	- Generate cryptographically secure challenge data for ``MRTD+IAS`` validation with sufficient entropy (to be used in Anti-Cloning Internal Authentication protocol by the Wallet Instance), storing it with an appropriate expiration time. Moreover, the MRTD PoP Service MUST ensure challenge uniqueness to prevent reuse attacks.
+	- Create a new unique ``nonce`` for the next step to prevent replay attacks.
+	- Validate session continuity by ensuring the ``auth_session`` parameter corresponds to an active session.
+	- Return HTTP *202 Accepted* status to indicate asynchronous processing initiation.
+	- Include proper Content-Type header (``application/json; charset=utf-8``).
+	- Handle service errors and return appropriate error responses.
+	- Extract and validate MRZ information if provided from external registry services.
+
+**The Wallet Instance MUST:**
+
+	- Validate HTTP response status (*202 Accepted*) and content type.
+	- Parse JSON response and validate required parameters (``challenge``, ``nonce``).
+	- Extract challenge data for cryptographic document validation.
+	- Store new ``nonce`` value securely for subsequent validation requests.
+	- Validate optional MRZ information if present in the response.
+	- Handle errors, providing relative user feedback.
+	- Store challenge data temporarily in secure memory (not persistent storage).
+	- Prepare NFC reading session.
+
+The Wallet Instance performs NFC-based electronic document reading and validation, then submits the evidence to the PID Provider for final verification and identity correlation with the LoA3 authentication result.
+
+MRTD PoP Validation Request
+"""""""""""""""""""""""""""
+
+Upon all the evidence has been collected through NFC interaction with the Electronic Document, the Wallet Instance MUST send all data to the PID Provider Authorization Server for the final validation and identity cross checks. The Wallet Instance MUST send a HTTP POST Request with ``application/json`` as content type, including Wallet Attestation and Wallet Attestation JWT PoP in the header according to *OAuth 2.0 Attestation Based Client Authentication*. The payload of the Request contains the following parameters.
```

Comment:
pls a massive replace using an href  on all the occurencies of *OAuth 2.0 Attestation Based Client Authentication*